src/components/MovieDetails/MovieDetails.js

```diff
@@ -69,6 +16,15 @@ export const MovieDetails = ({ location, user, ratings, setRatings }) => {
         <div className="overview-container">
           <label className="movie-overview">Overview</label>
           <p className="movie-description">{movie.overview}</p>
+          <iframe
+            title={movie.title + ' trailer'}
```

Comment:
Use a template literal here.

Reply:
That would definitely look cleaner, good call